packages/liquidator/test/index.js

```diff
@@ -163,8 +159,127 @@ contract("index.js", function(accounts) {
       assert.notEqual(spyLogLevel(spy, i), "error");
     }
 
-    // There should only be 1 log that communicates that bot is exiting early
+    // There should only be 1 log that communicates that bot is exiting early,
+    // which indicates that no approvals were sent.
+    assert.equal(spy.getCalls().length, 1);
+    assert.isTrue(spyLogIncludes(spy, 0, "expired"));
+  });
+
+  it("Post EMP expiry, liquidator can withdraw rewards but will not attempt to liquidate any undercollateralized positions", async function() {
```

Comment:
This is the new test: a liquidation is disputed and resolved post-expiry, test that the liquidator can withdraw rewards from it